### DISCUSSION TITLE
Slider rendering glitches in editor on specific beatmap

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#19992] [Q&A] [answered]
https://osu.ppy.sh/beatmapsets/1806319#osu/3704273

https://user-images.githubusercontent.com/53872073/187052808-f95921a4-d1fa-4f2b-b1c8-2e517621467e.mp4

I'm using Debian 11 on Intel UHD 630.

**Comment by peppy** [comment, marked as answer]
Most likely a different manifestation of https://github.com/ppy/osu/issues/7477.
